src/main/java/org/wso2/siddhi/plugins/idea/psi/AggregationNameNode.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Fix license year.

Reply:
@RAVEENSR check other places as well (where u have added new files)

Reply:
Fixed in the commit aca3f1f